packages/teleport-plugin-module-generator-angular/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@teleporthq/teleport-plugin-module-generator-angular",
```

Comment:
Based on our naming conventions, this should be: `teleport-plugin-angular-module-generator`